.github/workflows/release.yml

```
@@ -0,0 +1,124 @@
+name: Tag
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build and deploy release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        brand: ["payex", "swedbankpay"]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Fetch all history for all tags and branches
+        run: git fetch --prune --unshallow
+
+      # Set brand specific variables
+      - name: Environment variables
+        id: variables
+        run: |
+          chmod +x ./.github/scripts/variables.sh
+          ./.github/scripts/variables.sh
```

[`git update-index --chmod=+x ./.github/scripts/variables.sh`](https://stackoverflow.com/a/13593391/61818) pl0x.

```suggestion
        run: ./.github/scripts/variables.sh
```

---

Ah, very nice!